spec/result.json

```diff
@@ -0,0 +1,73 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
"draft".

The best we can do is to make this non-normative "for the assistance of developers".
(If there is a timescale for completion, I couldn't find it.)